controller/src/controller.rs

```diff
@@ -300,6 +320,20 @@ impl<T: BottlerocketShadowClient> BrupopController<T> {
                             event!(Level::INFO, name = %brs.name_any(), "Began updating new node.")
                         }
                     }
+                } else {
+                    event!(
+                        Level::INFO,
+                        "stop finding ready brs and complete remaining actions."
+                    );
+                }
+            } else {
+                // Bring one more node each time if the active nodes size is less than MAX_CONCURRENT_UPDATE setting.
+                if active_set_size < get_max_concurrent_update()? {
```

Comment:
Try to nest a bit less. Does clippy say anything?

```suggestion
            } else  if active_set_size < get_max_concurrent_update()? {
                // Bring one more node each time if the active nodes size is less than MAX_CONCURRENT_UPDATE setting.
               
```